state-sync/src/coordinator.rs

```diff
@@ -1716,23 +1773,85 @@ mod tests {
     }
 
     #[test]
-    fn test_process_chunk_request_message() {
+    fn test_invalid_chunk_request_messages() {
         // Create a coordinator for a validator node
         let mut validator_coordinator = test_utils::create_validator_coordinator();
 
-        // Create a test chunk request message
+        // Constants for the chunk requests
         let peer_network_id = PeerNetworkId::random();
-        let chunk_request_message = create_chunk_request_message(0, 0, 250, 10);
+        let current_epoch = 0;
+        let chunk_limit = 250;
+        let timeout_ms = 1000;
```

Comment:
shouldn't this be dictated by the config passed into the validator_coordinator?

Reply:
In real code, yes, but here we're constructing the chunk responses/requests manually for the unit tests and throwing them at the coordinator. If we wanted to apply these values first to the configs and then create coordinator objects that produce the chunk responses/requests, that would be more an e2e/integration style test, no? 😄

Reply:
I mean... this is defined in the config, right? that's how you generated the validator_coordinator? Maybe I might be missing something but isn't this how we got into the wonky state that we had to manually set 250 in the configs for testing?

Reply:
Synced offline 😄